csfieldguide/chapters/content/en/computer-graphics/sections/graphics-transformations.md

```diff
@@ -5,8 +5,13 @@ In fact, every pixel you see in an image has usually had many calculations made
 
 Let's start with some simple but common calculations that are needed in graphics programming.
 The following interactive shows a cube with symbols on each face.
-You can move it around using what's called a {glossary-link term="translation"}translation{glossary-link end}, which simply adjusts where it is placed in space.
-The camera will always focus on the cube, so to see the back of the cube move it behind the camera.
+You can move it around using what's called a {glossary-link term="translation"}translation{glossary-link end}, which simply adjusts where it is placed in space:
+
+- A positive or negative change in *x* will shift the box *right or left* respectively.
+- A positive or negative change in *y* will shift the box *up or down* respectively.
+- A positive or negative change in *z* will shift the box *in or out* respectively.
+
+The camera will always focus on the cube, so to see the back of the cube move it behind the camera with a positive *z* translation.
```

Comment:
I think these changes also sufficiently address the second bullet point in the issue:

> The first interactive is quite confusing when you start using it, because of how the camera moves ...

The only thing I'd suggest is just putting "always focus on the cube" in bold text. I think the new text that's been added here will make the interactive make more sense, so the interactive probably wont need any changes